SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentReader.cpp

```diff
@@ -98,6 +102,49 @@ ComponentReader::ComponentReader(USpatialNetDriver* InNetDriver,
 {
 }
 
+template<typename T>
+void ForEachUnique(const TArray<T>& ArrayA, const TArray<T>& ArrayB, TFunction<void(const T Value)> Func)
```

Comment:
Do we still need this in the latest iteration?

Reply:
nope removing.